pkg/releasepayload/jobrunresult/helpers.go

```diff
@@ -18,3 +18,31 @@ func (in ByCoordinatesName) Len() int {
 func (in ByCoordinatesName) Swap(i, j int) {
 	in[i], in[j] = in[j], in[i]
 }
+
+type ByCompletionTime []v1alpha1.JobRunResult
```

Comment:
i don't see this being used.

Reply:
It's not, but I added it anyway just in-case we want/need to use it later...
It doesn't work in this scenario because we don't want to wait for the jobs to "Complete" to order them correctly.